app/views/login.html

```diff
@@ -68,14 +68,15 @@
 				<!--</div>-->
 				<div class="form-group" >
 					<span class="login-texts" >Login Id</span>
-					<input type="text" class="form-control input input-sm login-inputs" data-ng-model="user.username" name="username" placeholder="Username"  autocomplete="new-password" >
+					<input type="text" class="form-control input input-sm login-inputs" data-ng-model="user.username" name="username" placeholder="Username">
 
 				</div>
+<!--	changes for disbaling autocomplete for login-->
 				<div class="form-group" >
 					<span class="login-texts">Password</span>
-					<input name="password0" type="password"  value="Dummy" style="display: none" />
-					<input type="password" class="form-control password-input input input-sm login-inputs" id="divID" data-ng-model="user.password"  name="password" placeholder="Password"	 autocomplete="new-password" value="Dummy">
-					<input name="password1" type="password"  value="Dbmnmummy" style="display: none" />
+<!--					<input name="password0" type="password"  value="Dummy" style="display: none" />-->
```

Comment:
if not required, please delete the commented code. We can always retrieve these back from the history